src/nimgen/c2nim.nim

```diff
@@ -87,7 +91,7 @@ proc c2nim*(fl, outfile: string, c2nimConfig: c2nimConfigObj) =
     if file.isAbsolute():
       passC &= "const header$# = \"$#\"\n" % [fname, file]
     else:
-      passC &= "const header$# = sourcePath & \"$#\"\n" %
+      passC &= "const header$# = sourcePath / \"$#\"\n" %
         [fname, file.relativePath()]
```

Comment:
Was this broken before?

Reply:
Yes, at least on Linux, but probably on others too.

OLD:
```nim
sourcePath & "foo.h"  # /somefolderfoo.h
```
NEW
```nim
sourcePath / "foo.h"  # /somefolder/foo.h
```

Reply:
Probably cause most wrappers used absolute paths and didn't go through this else.